docs/examples/efficientnet/model_repository/efficientnet_ensemble/config.pbtxt.in

```diff
@@ -0,0 +1,66 @@
+# The MIT License (MIT)
```

Comment:
There should be `1` created next to the config.

Reply:
Done